ops/dockerfiles/frontend.Dockerfile

```diff
@@ -4,10 +4,13 @@ WORKDIR mycro
 
 RUN npm install -g truffle
 
+# used for prod environment
+RUN npm install -g serve
```

Comment:
can this be added to line 5? `RUN npm install -g truffle serve`?

Reply:
it could be. I'm not convinced it needs to be though? I would argue having it separate with a comment improves readability of the file especially since serve is only used in prod and not in dev I could imagine someone being confused why it's being installed

Reply:
Just saw your comment below about each RUN adding layers and making the image bigger... I see the argument for combining the commands now. Have we been hitting space issues with images?